summer16short.md

```diff
@@ -75,17 +77,7 @@ We are committed to being fully transparent about how we make and spend money. I
 - We welcome students with a broad array of technical experiences--no coding experience is required, but a basic comfort level with technology is preferred.
 - B.Y.O.Laptop (Mac / PC / Linux)
 
-
```

Comment:
I think this will change the order of things on the page, but I'm not concerned about it.

Reply:
Agreed—if this is seems overly restrictive in future we can refactor the layout & includes to allow more flexibility.